status/status_test.go

```diff
@@ -29,6 +29,7 @@ import (
 	apb "github.com/golang/protobuf/ptypes/any"
 	dpb "github.com/golang/protobuf/ptypes/duration"
 	"github.com/google/go-cmp/cmp"
+
```

Comment:
Now that we reverted some of your older changes, this file contains a single whitespace change. It would be better to undo this. Sorry for the trouble. Thanks.